app/src/main/java/org/oppia/android/app/databinding/AppCompatCheckBoxBindingAdapters.java

```diff
@@ -0,0 +1,18 @@
+package org.oppia.android.app.databinding;
+
+import android.content.res.ColorStateList;
+import androidx.annotation.ColorInt;
+import androidx.annotation.NonNull;
+import androidx.appcompat.widget.AppCompatCheckBox;
+import androidx.databinding.BindingAdapter;
+
+/**
+ * Custom data-binding adapters for {@link AppCompatCheckBox}s.
+ */
```

Comment:
```suggestion
/** Custom data-binding adapters for {@link AppCompatCheckBox}s. */
```

Reply:
Okay ben

Reply:
![image](https://user-images.githubusercontent.com/92685493/221245578-8c4a3e26-467e-437c-bca2-1ff8e1be749f.png)
@BenHenning It's showing this after converting it to a single line.

Reply:
Ah, I forgot that checkstyle doesn't allow for this (valid per the Google Java style guide which we follow) version--thanks for checking!

Reply:
Thanks ben.